src/style.css

```diff
@@ -0,0 +1,78 @@
+[data-qa="big"] {
```

Comment:
get used to style all elements using classes
and check this point from [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)
![Screenshot from 2023-04-06 13-23-02](https://user-images.githubusercontent.com/33029864/230351261-958fb110-1033-4c49-97dc-8182cb805c81.png)